### ISSUE TITLE
[steamshovel] invalid section header in docstrings (Trac #1735)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1727]
unfortunately `Authors` is not a valid section header for numpy style docstrings. A list of valid headers is available from https://pypi.python.org/pypi/sphinxcontrib-napoleon 

Sphinx gives the following error:

```text
/Users/kmeagher/icecube/combo/release/lib/icecube/steamshovel/artists/util.py:docstring of icecube.steamshovel.artists.util.jacobian:28: SEVERE: Unexpected section title.

Authors
-------
/Users/kmeagher/icecube/combo/release/lib/icecube/steamshovel/artists/util.py:docstring of icecube.steamshovel.artists.util.propagate_covariance:33: SEVERE: Unexpected section title.

Authors
-------

```

Migrated from https://code.icecube.wisc.edu/ticket/1735
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "unfortunately `Authors` is not a valid section header for numpy style docstrings. A list of valid headers is available from https://pypi.python.org/pypi/sphinxcontrib-napoleon \n\nSphinx gives the following error:\n{{{\n/Users/kmeagher/icecube/combo/release/lib/icecube/steamshovel/artists/util.py:docstring of icecube.steamshovel.artists.util.jacobian:28: SEVERE: Unexpected section title.\n\nAuthors\n-------\n/Users/kmeagher/icecube/combo/release/lib/icecube/steamshovel/artists/util.py:docstring of icecube.steamshovel.artists.util.propagate_covariance:33: SEVERE: Unexpected section title.\n\nAuthors\n-------\n\n}}}", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "combo core", 
    "summary": "[steamshovel] invalid section header in docstrings", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2016-06-10T07:31:38", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160615T18:40:44: olivas changed component from "cmake" to "icecube offline"

**Comment by icecube-trac** [comment]
Trac update at 20160621T13:13:34: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski commented:

> In changeset 147640:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="147640"
> closes #1727
> ```


* hdembinski changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas commented:

> Milestone August 2016 deleted


* olivas changed milestone from "August 2016" to ""
